data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -34,7 +34,7 @@ done
 
 echo "Gathering bootstrap journals ..."
 mkdir -p "${ARTIFACTS}/bootstrap/journals"
-for service in approve-csr bootkube crio crio-configure ironic kubelet master-bmh-update release-image release-image-download sssd
+for service in approve-csr bootkube crio crio-configure ironic kubelet master-bmh-update release-image release-image-download sssd bootstrap-verify-api-server-urls
```

Comment:
@patrickdillon @jstuever Do you think we need this?

Reply:
I *think* so

Reply:
I don't think this is a service? It's just a file that has some important functions and is sourced by the bootkube? I'm thinking these echo statements should show up in the bootkube.log since it's the bootkube service calling these functions?

Most services are declared [here](https://github.com/openshift/installer/tree/master/data/data/bootstrap/systemd/units)